Ginger/GingerCoreCommon/Repository/RepositoryItemBase.cs

```diff
@@ -1076,6 +1073,18 @@ public void SetDirtyStatusToNoChange()
             DirtyStatus = eDirtyStatus.NoChange;
 
             // Properties
+            //foreach (PropertyInfo PI in this.GetType().GetProperties().Where(pi =>
+            //    pi.GetCustomAttribute(typeof(IsSerializedForLocalRepositoryAttribute)) != null
+            //        && typeof(IObservableList).IsAssignableFrom(pi.PropertyType)
+            //            && (pi.GetValue(this) as IObservableList) != null))
+            //{
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5936174)